src/utils/local.ts

```diff
@@ -104,15 +107,46 @@ export class LocalRelease extends EventEmitter implements Release {
     if (path.endsWith(configFile)) {
       const conf = this.getConfig()
       this.config = conf.config
-      this.files = conf.files || []
       this.app = conf.app || conf.app_id || ""
+      this.getFiles(conf.files)
     } else if (path.endsWith(secretsFile)) {
       this.secrets = this.getSecrets()
     }
     this.emit('update', this)
   }
+
+  private getFiles (files: string[] | undefined) {
+    if (!files) return
+
+    const conf = this.getConfig()
+    if (conf.files) conf.files.forEach((file) => {
+      if (!this.isIn(this.files, file)) this.files.push(file)
+    })
+
+    files.forEach((file) => {
+      glob(file, {}, (error, filesFound) => {
```

Comment:
You will want to use the `cwd:` option for `glob` to specify the `cwd` passed to `LocalRelease`. This setting needs to propagate anywhere interacting with the current working directory.

Reply:
Yup - implemented that when writing the tests :)